README.md

```diff
@@ -293,12 +293,12 @@ You can use the following options at build time to influence the build. During t
 |png|`--with-png`|Whether to build with libpng support|Auto|
 |gcov|`--enable-gcov`|Build gcov coverage tests|No|
 |tests|`--enable-tests`|Build tests (requires `bash`)|No|
-|python2|`--enable-python`|Build Python library integration|No|
+|python3|`--enable-python3`|Build Python library integration|No|
```

Comment:
i'd drop the 3 here

Reply:
This just reflects the change. But I would prefer the second one to just say python if I figure out which place needs to be change back.

Reply:
yeah i'd prefer it just be `--enable-python`

Reply:
So I think this actually comes from  `setup.py` under setup at the top. I'll try giving it a quick spin using just `--enable-python`